soccer/gameplay/robocup-py.cpp

```diff
@@ -326,6 +500,26 @@ BOOST_PYTHON_MODULE(robocup)
 	class_<Geometry2d::Circle, bases<Geometry2d::Shape> >("Circle", init<Geometry2d::Point, float>())
 		.def("intersects_line", &Circle_intersects_line)
 		.def("nearest_point", &Geometry2d::Circle::nearestPoint)
+        .def("contains_point", &Geometry2d::Circle::containsPoint)
+        .def("center", &Circle_get_center)
```

Comment:
The wrapper is there because we can't expose fields (I tried and got a bunch of errors?). I can move the wrapper function to the circle class if that makes more sense...